### ISSUE TITLE
Feature Request: Sign in with Keybase

### ISSUE THREAD

**Issue by RyanSinger** [keybase/client#5315]
Hi!

It's would be really cool if there was easy UX for my users to sign onto a web application with Keybase. I'm thinking something along the lines of a chrome extension that is a device with a device key, that can sign statements in a challenge-response model.

Just a thought.

**Comment by cjb** [comment]
Yeah!  We think so too.  @jzila prototyped such a Chrome extension two years ago:

https://github.com/jzila/kb-login-ext

Another possible implementation would be having the website write a challenge to:
> /keybase/private/somewebsite.com,ryansinger/login

which you then sign.

**Comment by RyanSinger** [comment]
These are both close to what I want. Unfortunately, the linked chrome extention uses the saved PGP private key, which I wish was deprecated. I want it to be it's own device in the sigchain.

The private directory approach does that well, but requires going to the terminal to sign something.

**Comment by cjb** [comment]
@RyanSinger Yeah, sounds like you'd prefer a Chrome extension that talks to the service running on your machine over RPC.  (Or runs its own service inside the extension.)

Or to have the KBFS signature requests exposed in the GUI so you don't need to go to the terminal to sign them.

Thanks, CC @malgorithms for the feature upvotes!
